docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+        POSTGRES_USER: uclapi_default
+        POSTGRES_PASSWORD: test
+  cache:
+    image: postgres
+    restart: always
+    ports:
+      - "5432"
+    environment:
+        POSTGRES_USER: uclapi_gencache
+        POSTGRES_PASSWORD: test
+  redis: 
+    image: redis
+    restart: always
+    ports: 
+      - "6379"
+  web:
+    build:
+      context: .
+      dockerfile: dev.Dockerfile
+    volumes:
+      - .:/uclapi
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+      - cache
+      - redis
+    env_file: ./backend/uclapi/.env
+    environment: 
+        DOCKER: "yes"
+        DOCKER_DEPLOYMENT: "dev"
+        DB_UCLAPI_NAME: uclapi_default
+        DB_UCLAPI_HOST: db
+        DB_UCLAPI_USERNAME: uclapi_default
+        DB_UCLAPI_PASSWORD: test
```

Comment:
do we **need** to set these passwords?

Reply:
Yeah it's for a local dev deployment /cc @Maaarcocr

Reply:
For my own local setup I don't use any password. It kinda just works. Unless this a docker specific thing?

Reply:
I guess we may not need them. When I wrote that I was following the .env example that said to put my password there.